plugins/quickinspector/quickinspector.cpp

```diff
@@ -217,6 +223,48 @@ static QString qsgMaterialFlagsToString(QSGMaterial::Flags flags)
     return list.join(QStringLiteral(" | "));
 }
 
+static QString qSGRenderNodeStateFlagsToString(QSGRenderNode::StateFlags flags)
```

Comment:
The flags/enums stringification functions are doable more elegantly nowadays using metaenum.h, avoiding a lot of the duplication.